modules/101-cert-manager/openapi/config-values.yaml

```diff
@@ -35,37 +35,73 @@ properties:
     x-examples:
     - [{"key":"dedicated.deckhouse.io","operator":"Equal","value":"cert-manager"}]
 
-  cloudflareAPIToken:
-    type: string
-    x-examples: ["token"]
-    description: |
-      [API Tokens](https://cert-manager.io/docs/configuration/acme/dns01/cloudflare/#api-tokens) allow application-scoped keys bound to specific DNS zones.
-
-      API Tokens are recommended for higher security, since they have more restrictive permissions and are more easily revocable.
-
-      It allows you to verify that domains specified
-      in the Certificate resource are managed by `cert-manager` and kept by the Cloudflare DNS provider.
-      Verification is performed by adding special TXT records for
-      the [ACME DNS01 Challenge Provider](https://cert-manager.io/docs/configuration/acme/dns01/) domain.
-
-  cloudflareGlobalAPIKey:
-    type: string
-    description: |
-      The Cloudflare Global API key for managing DNS records
-
-      It allows you to verify that domains specified
-      in the Certificate resource are managed by `cert-manager` and kept by the Cloudflare DNS provider.
-
-      Verification is performed by adding special TXT records for
-      the [ACME DNS01 Challenge Provider](https://cert-manager.io/docs/configuration/acme/dns01/) domain.
-    x-examples: ["key"]
+#  cloudflareAPIToken:
+#    type: string
+#    x-examples: ["token"]
+#    description: |
+#      [API Tokens](https://cert-manager.io/docs/configuration/acme/dns01/cloudflare/#api-tokens) allow application-scoped keys bound to specific DNS zones.
+#
+#      API Tokens are recommended for higher security, since they have more restrictive permissions and are more easily revocable.
+#
+#      It allows you to verify that domains specified
+#      in the Certificate resource are managed by `cert-manager` and kept by the Cloudflare DNS provider.
+#      Verification is performed by adding special TXT records for
+#      the [ACME DNS01 Challenge Provider](https://cert-manager.io/docs/configuration/acme/dns01/) domain.
+#
+#  cloudflareGlobalAPIKey:
+#    type: string
+#    description: |
+#      The Cloudflare Global API key for managing DNS records
+#
+#      It allows you to verify that domains specified
+#      in the Certificate resource are managed by `cert-manager` and kept by the Cloudflare DNS provider.
+#
+#      Verification is performed by adding special TXT records for
+#      the [ACME DNS01 Challenge Provider](https://cert-manager.io/docs/configuration/acme/dns01/) domain.
+#    x-examples: ["key"]
+#
+#  cloudflareEmail:
+#    type: string
+#    format: email
+#    description: |
+#      The email used for accessing the Cloudflare platform.
+#    x-examples: ["example@example.com"]
```

Comment:
Conversion code (conversion.go) is commented but config-values scheme is changed. Will it lead to a disaster?